doc/Messaging.md

```diff
@@ -0,0 +1,148 @@
+# Vitess Messaging
+
+# Overview
+
+Vitess messaging gives the application an easy way to schedule and manage work that needs to be performed asynchronously. Under the covers, messages are stored in a traditional MySQL table and therefore enjoy the following properties:
+
+* **Transactional**: Messages can be created or acked as part of an existing transaction. The action will complete only if the commit succeeds.
```

Comment:
This one is not the main attraction, I would move it down, keep Scalable as the first one in the list.